src/nvim/generators/gen_unicode_tables.lua

```diff
@@ -250,9 +250,13 @@ local build_emoji_table = function(ut_fp, emojiprops, doublewidth, ambiwidth)
         table.insert(emoji, { n, n_last })
       end
 
-      -- Characters below 1F000 may be considered single width traditionally,
-      -- making them double width causes problems.
-      if n >= 0x1f000 then
+      -- Special cases:
+      --   * Below 1F000 is considered single-width traditionally.
+      --   * Regional Indicator Symbols 0x1F1E6-0X1F1FF are almost always used
+      --     in pairs to represent a country flag, so special-case as
+      --     single-width to avoid 4 virtual columns. #16447
+      -- Make an exception for the Regional Indicator Symbols 0x1F1E6-0X1F1FF
```

Comment:
this line now is not needed

Reply:
Sorry, I just used the Github interface to commit the documentation patch you had earlier suggested.  Github did the rest and I didn't look closely enough at the merge.  I've updated that commit, to remove the unneeded line.